core/dbt/tests/fixtures/project.py

```diff
@@ -282,8 +282,8 @@ def adapter(
     project_root,
     profiles_root,
     profiles_yml,
-    dbt_project_yml,
     clean_up_logging,
+    dbt_project_yml,
```

Comment:
At first I was confused about why this fixture argument moved. At first I thought order of operations was important for some reason here. Looking through the commits though, I think what happened is that it got removed in one commit, and then re-added in another.